platform/ios/PolyPodApp/PolyPod/PolyStyle.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+struct PolyStyle {
+    struct Spacing {
+        static let plSpace1x = 4.0
+        static let plSpace2x = {
```

Comment:
Er. Does this need to be variables with the names? Can't it be a dictionary?

Reply:
It is much easier to access them like this. They are basically types, and we have nice autocompletion for them. We don't get that with dictionaries.

Reply:
Still, trading convenience for DRY... I don't know.

Reply:
The overall styling approach would probably change, we would want to have a single configuration, say a JSON file that is loaded in both iOS and Android platforms.